comms/src/connection/net_address/net_addresses.rs

```diff
@@ -12,12 +12,16 @@ pub const MAX_CONNECTION_ATTEMPTS: u32 = 3;
 #[derive(Debug, Clone, Deserialize, Serialize, PartialEq, Default)]
 pub struct NetAddresses {
     pub addresses: Vec<NetAddressWithStats>,
+    last_attempted: Option<DateTime<Utc>>,
```

Comment:
Maybe the struct name should now change to "NetAddressesWithStats"?

Reply:
Done